run_viralrecon.sh

```diff
@@ -46,31 +46,29 @@ fi
 echo "$(date) : Running viralrecon"
 #export SINGULARITY_CACHEDIR=/home/pgupta/singularity
 #export NXF_SINGULARITY_CACHEDIR=/home/pgupta/singularity
+# export APPTAINER_CACHEDIR=/home/pgupta/apptainer
+# export NXF_APPTAINER_CACHEDIR=/home/pgupta/apptainer
 
 #Use UPHL_viralrecon.config to update Pangolin container, if needed
 
 if [ "$resume_option" = "-resume" ]
 then
-    nextflow run nf-core/viralrecon -name ${run_name} -resume ${run_name} --input $out_dir/$infile \
+    nextflow run nf-core/viralrecon -resume ${run_name} --input $out_dir/$infile \
                                     --primer_set_version 5.3.2 \
                                     --outdir $out_dir \
-                                    --nextclade_dataset false \
-                                    --nextclade_dataset_tag false \
                                     --schema_ignore_params 'genomes,primer_set_version' \
-                                    --multiqc_config $script_dir/conf-files/new_multiqc_config.yaml \
                                     -profile singularity \
+                                    --multiqc_config $script_dir/conf-files/new_multiqc_config.yaml \
                                     -params-file $script_dir/conf-files/UPHL_viralrecon_params.yml \
                                     -c $script_dir/conf-files/UPHL_viralrecon.config \
                                     -w $work_dir
 else
 nextflow run nf-core/viralrecon --input $out_dir/$infile \
                                 --primer_set_version 5.3.2 \
                                 --outdir $out_dir \
-                                --nextclade_dataset false \
-				                --nextclade_dataset_tag false \
                                 --schema_ignore_params 'genomes,primer_set_version' \
-                                --multiqc_config $script_dir/conf-files/new_multiqc_config.yaml \
                                 -profile singularity \
+                                --multiqc_config $script_dir/conf-files/new_multiqc_config.yaml \
                                 -params-file $script_dir/conf-files/UPHL_viralrecon_params.yml \
                                 -c $script_dir/conf-files/UPHL_viralrecon.config \
```

Comment:
The only difference I see between these is the `-resume` flag is added when resume_option" = "-resume". Is there anything else?

Reply:
Yes, that's the only difference.